index.html

```diff
@@ -25,7 +25,7 @@ <h1>Hello, Welcome to my Portfolio!</h1>
       <h1 class="project-tile">My Projects</h1>
```

Comment:
- [ ] As suggested by the previous reviewer, Kindly make sure you have given a reason that is concise for using alt. Alt is used for images so that visually impaired users can know what is in the position of that image as well as for screen readers. You can write a review for this like "Kindly add Alt to the image tag to provide alternative information for the visually impaired users to know what is the position of the image as well as for screen readers."